### PR TITLE
Update password setup email template

### PR DESCRIPTION
**Pull request by RahulJha-mastek**
Story AB#14839
Task AB#15284

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+      "htmlContent": "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\"><html xmlns=\"http://www.w3.org/1999/xhtml\"><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\" /><title></title></head><body><p>You requested a password reset for your Buying Catalogue account.</p><p>Please use <a href=\"{0}\">this link</a> to reset your password.</p><p>The link can be used only once and will be valid for 24 hours from the time you received this email.</p><p>Your new password must be at least ten characters in length and match on confirmation.</p><p>It must contain at least 3 of the following 4 character types:</p><p>lower case letters (a-z)<br />upper case letters (A-Z)<br />numbers (0-9)<br />special characters (e.g. !@#$%^&amp;*)</p><p>Yours sincerely,<br />The Buying Catalogue team</p></body></html>",
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json:14]: It looks like you've added some unnecessary white space to the end of this line.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
...
...you can now set your password.\n\n{0}\n\nYour password must be at least ten characters in length and match on confirmation.\n\nIt must contain at least 3 of the following 4 character types:\n\nlower case letters (a-z)\nupper case letters (A-Z)\nnumbers (0-9)\nspecial characters (e.g. !@#$%^&*)\n\nThe link can be used only once to set up one account, and will be valid for 24 hours from the time you received this email.\n\nHelp us improve the Buying Catalogue\n\nWe would really like your help in shaping and influencing the development of the Buying Catalogue. You can support us in two ways:\n\n\tBy participating in user research. If you'd like to work with us on our research, please email buying.catalogue@nhs.net.Once we receive your email, we'll invite you to join our user research sessions where we'll show new features and get your feedback on them.\n\n\tBy providing feedback to us on your experience of using the Buying Catalogue. We have a Buying Catalogue user survey, that you can access directly at https://forms.office.com/Pages/ResponsePage.aspx?id=Hwf2UP67GkCIA2c3SOYp4nDHKEWnXcFHiqdJhf0fCJtUNDNFRUFZVFU5RkRQTEpWU0RQVlVXMUpRQi4u \n\nYours sincerely,\n\nThe Buying Catalogue team"
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json:26]: For the plain text version, recommend that `and you can now set your password` becomes `and you can now set your password using the link below.`

Also in the plain text version of `please email buying.catalogue@nhs.net.Once` there is a space missing before `Once`.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Just a couple of small changes requested.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json (hunk 1 above)
Comment (RahulJha-mastek) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json]: Removed whitespace.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json (hunk 2 above)
Comment (RahulJha-mastek) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/appsettings.json]: Updated content to below 
`and you can now set your password using the link below.`
and 
`please email buying.catalogue@nhs.net. Once`

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**